doc/classes/ClassDB.xml

```diff
@@ -123,6 +135,7 @@
 			<argument index="2" name="no_inheritance" type="bool" default="false">
 			</argument>
 			<description>
+				Return whether 'class' or its ancestry if 'no_inheritance' is false, has a method called 'method' or not.
```

Comment:
You shouldn't put a comma before the verb like that as it breaks the sentence. Instead, you can put the whole "or its ancestry if 'no_inheritance' is false" between commas:
"Return whether 'class'**,** or its ancestry if 'no_inheritance' is false**,$$ has a method called 'method' or not."

Reply:
what is the rationale behind this, i would rather use parenthesis, since thats the correct way
like:
> Return whether 'class'  (or its ancestry, if 'no_inheritance' is false) has a method called 'method' or not.

Reply:
https://www.grammarly.com/blog/comma/

>Commas with Interrupters or Parenthetical Elements
>
>Interrupters are little thoughts that pop up in the middle of a sentence to show emotion, tone, or emphasis. A parenthetical element is a phrase that adds extra information to the sentence but could be removed without changing the meaning of the sentence. Both interrupters and parenthetical elements should be set off with commas.